tests/rpmdb.at

```diff
@@ -58,16 +58,14 @@ RPMTEST_CLEANUP
 
 AT_SETUP([rpm -qa 3])
 AT_KEYWORDS([rpmdb query])
-RPMTEST_SETUP
+RPMDB_INIT
 
 RPMTEST_CHECK([
 runroot rpm -U --nodeps --ignorearch --ignoreos --nosignature \
 	/data/RPMS/foo-1.0-1.noarch.rpm \
 	/data/RPMS/hello-2.0-1.x86_64-signed.rpm
 
-# Assert that there are at least 3 packages in the rpmdb (we are using that of
-# the test image here so that is pretty much guaranteed).
-runroot rpm -qa | wc -l | xargs test 2 -lt || exit 1
+runroot rpm -qa | wc -l | xargs test 2 -eq || exit 1
```

Comment:
This was actually deliberately > 2 :sweat_smile:, so that we could test that filtering out the package names only shows those (and not any other). But meh...

Reply:
It's probably better to just install 3 packages instead of 2, I was just too lazy to do that I guess...

Reply:
Right, a subtlety I missed there. We can just toss more packages into the db, not a problem.

Reply:
Added two more packages to the install and test for the exact expected number of 'em in the db since we now can.

Reply:
Thanks, it's funny that *that* was easier than writing a whole comment line with an excuse :laughing:

Reply:
Yup, happens every now and then: one goes to great lengths trying to avoid a thing where actually just doing the it would be far, far easier :sweat_smile: